core/src/main/java/io/paradoxical/cassieq/dataAccess/QueueRepositoryImpl.java

```diff
@@ -231,10 +236,12 @@ private boolean upsertQueueDefinition(@NonNull final QueueDefinition queueDefini
         final int currentVersion = currentQueueDefinition.getVersion();
         final int nextVersion = currentVersion + 1;
 
+        final QueueSizeCounterId newQueueCounterId = getUniqueQueueCounterId(currentQueueDefinition.getQueueName(), nextVersion);
```

Comment:
queue definition has a .getQueueId(), lets just pass that into the function for use (want to avoid `QueueId.valueOf(blah)`)